index.html

```diff
@@ -21,26 +21,102 @@ <h2 class="notoc">Contents</h2>
+  <li>When new features are added after a document has already received wide
+  review (perhaps requesting a review limited to the new features).</li>
+  </ul>
+  </section>
 
 <section id="who_to_ask_for_review">
-<h2>Who to ask for review?</h2>
- 
-<p>Much of this document focuses on how and when to conduct horizontal reviews, but they are only a subset of a full wide review, which must also include other stakeholders including Web developers, technology providers and implementers not active in the Working Group, external groups and standards organizations working on related areas, etc. Here is a list of suggestions:</p>
+<h2>Who to ask for wide review?</h2>
+
+ <p>The objective is to ensure that the entire set of stakeholders of the Web
+ community, including the general public, have  adequate notice of the progress
+ of the Working Group and are able to actually perform reviews of and provide
+ comments on the specification. When considering <a
+ href="https://www.w3.org/Guide/transitions">requests to advance the maturity
+ level of the document</a>, the Director will consider who has been explicitly
+ offered a reasonable opportunity to review the document.</p>
+
+<p>You must publish an updated technical report, with the Status of the Document
+indicating clearly that you're looking for <em>wide review</em>, before inviting
+review from other stakeholders. Our <a
+href='https://github.com/w3c/transition-notifier/blob/main/notify.js'>tooling</a>
+monitors publications and propagates notifications to <a
+href="mailto:public-review-announce@w3.org">public-review-announce@w3.org</a>
+automatically (for example, <a
+href='https://lists.w3.org/Archives/Public/public-review-announce/2021Jun/0008.html'>Candidate
+Recommendation Snapshot: Payment Request API (Call for Wide Review)</a>).</p>
+
+<p>You should then reach out to:</p>
+
 <ul>
-<li> Groups listed in the WG's charter, especially those who manage dependencies.</li>
-<li> Groups jointly responsible for a particular document (if any)  (duh!).</li>
-<li> All horizontal groups (listed below).  If it appears that one of those is not relevant (and is not listed in your charter), talk to them informally to confirm that.</li>
-<li> Other groups, at your discretion, even if not in the WG charter, including other W3C groups, external organizations and SSOs, implementers, application developers, etc.</li>
-<li>The general public. Consider using blog posts, social media, or other ways of alerting the public to your document and requesting feedback.</li>
+<li>the groups listed in the WG's charter, especially those who manage
+dependencies.</li>
+<li>the groups jointly responsible for a particular document (if any).</li>
+<li>the <a href="#how_to_get_horizontal_review">horizontal groups</a>.  If it
+appears that one of those is not relevant (even if not listed in your charter),
+talk to them informally to confirm that. <em>Note that sending an email to the
```

Comment:
Not sure informal contact is adequate here for demonstrating HR as per the process? Isn't a "paper" trail needed that shows that the WG told the HR group about the change and the HR group said they had no comments?

Reply:
Be fault, the paper trail is expected but the Director did use judgment in the past. For example, we don't really expect an Accessibility review on the WebAssembly Core specification. @swickr , wdyt?

Reply:
If i18n feels that a review is needed, they will ask the WG to start the formal process.  If they agree that it's not needed, then we'd like to avoid unnecessary paperwork or process steps (mainly because we want to preserve as much bandwidth as possible).

Reply:
The Process does not seem to allow for WG's _not_ requesting review. I understand from the HR group's perspective that they don't want unnecessary work; on the other hand, from a WG's perspective, they want to have a formal statement of "we did what we were required to do". 

I wonder: is it actually less work to have the informal conversation than to review the formal request and respond with a "go ahead, no input from us"?

Reply:
Nigel, i think this process related suggestion doesn't belong in this PR. Could you raise an issue?

Reply:
Richard, I don't think I'm suggesting any change to the Process, I'm just trying to get the wording in this PR to match what folk need to do to meet the process as it stands. Maybe we need to wait for @swickr to respond to [@plehegar 's clarification question](https://github.com/w3c/documentreview/pull/21#discussion_r662476772) before we can proceed.

Reply:
I concur here that we should set proper expectation on the amount of paper trail needed. It's easier for the Director to conclude that no review was needed by i18n when there is an actual record from i18n, than relying on out of band conversation and uncheckable assertions. So, let's wait for @swickr input.

Reply:
> It's easier for the Director to conclude that no review was needed by i18n when there is an actual record from i18n, than relying on out of band conversation and uncheckable assertions.

Yep; explicitly recorded assertions are much preferred; thanks.

Reply:
Addressed in https://github.com/w3c/documentreview/pull/21/commits/89e323b984966f3cee7c8b5de5d128b48fcc4fef

Reply:
Just a nit: i wrote 'process' with a lowercase 'p', ie. referring to the general process, rather than the Process document.  Probably doesn't change anything.

Reply:
The Process does not require sign-off from every HRG. It requires it to transition without the Director's permission, but the Director can use their judgement to approve a transition that skips certain HRG reviews (such as the unnecessary a11y review of WebAssembly mentioned earlier).

Reply:
When filing the transition request without the full set of HRG reviews, groups should probably explain why they think a particular HRG's review is unnecessary, and the Director can accept when it seems obvious and push back if he feels it's inappropriate or borderline. I think that would be a reasonable substitute to asking the HRGs to do more paperwork on things that are clearly irrelevant to their interests.